src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -288,6 +288,21 @@ public function where($column, $operator = null, $value = null, $boolean = 'and'
         return $this;
     }
 
+    /**
+     * Add a basic where Like clause to the query, and return the All Like results.
+     *
+     * @param  \Closure|string|array|\Illuminate\Database\Query\Expression  $column
+     * @param  mixed  $value
+     * @return \Illuminate\Database\Eloquent\Model|static|null
+     */
+    
+    public function whereLike($column, $value = null)
+    {
+        $this->where($column,'Like','%'.$value.'%');
```

Comment:
Using double% to match directly, isn't there a performance problem?

Reply:
I don't think so, but let's wait for other opinion